lookup_plugins/azure_keyvault_secret.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: azure_keyvault_secret
+    author:
+        - Hai Cao <t-haicao@microsoft.com>
+    version_added: 2.7
+    requirements:
+        - requests
+        - azure
+        - msrest
+    short_description: read secret from Azure Key Vault.
+    description:
+      - This lookup returns the content of a secret kept in Azure Key Vault.
+    options:
+        _terms:
+            description: secret name of the secret to retrieve, version can be included like secret_name/secret_version.
+            required: True
```

Comment:
maybe all descriptions should start from uppercase and just put them into new line wit "-" for consistenct